### ISSUE TITLE
Update Optimize Intervention plumbing in HMI + pyciemss-service

### ISSUE THREAD

**Issue by YohannParis** [DARPA-ASKEM/terarium#3995]
Blocked by #3993 and TA3 design implemented in [#97](https://github.com/DARPA-ASKEM/pyciemss-service/issues/97)

Once the new `InterventionPolicy` is in place, we need to update the controller to update Simulation, Calibration, and Optimization requests to contain the new _InterventionPolicies_.

The _Requests_ should point to a list of `InterventionPolicy` ids, and the controller/services must provide them with the request based on `pyciemss-service` API.

**Tasks**
- [x] Update `SimulationRequest` to provide InterventionPolicies
- [x] Update `CalibrationRequestCiemss` to provide InterventionPolicies
- [ ] Update `OptimizeRequestCiemss` to provide InterventionPolicies

**Comment by Tom-Szendrey** [comment]
renaming task as simulation + calibration was done via another ticket.
